### PR TITLE
Check groups every time restserver start

### LINKED ISSUE #3451: [BUG] VC nni show in virtual clusters list but failed when update VC for user.
VC nni show in virtual clusters list but failed when update VC for user.

image

image

### PR DESCRIPTION
**Pull request by mzmssg**
Fix #3451

### CODE REVIEW

File: src/rest-server/src/models/v2/group.js [hunk 1]
```diff
@@ -355,8 +357,8 @@ const deleteNonexistVCs = async () => {
       if (groupItem.extension && groupItem.extension.acls && groupItem.extension.acls.virtualClusters) {
         const checkedVCs = groupItem.extension.acls.virtualClusters.filter((vcname) => vcSet.has(vcname));
         if (checkedVCs.length !== groupItem.extension.acls.virtualClusters.length) {
-          logger.info(`Update group: [${groupItem.groupname}] vc list. 
-          old: ${groupItem.extension.acls.virtualClusters}, new: ${checkedVCs}.`);
+          logger.info(`Update group: ${groupItem.groupname} vc list, ` +
```
Comment (abuccts) [inline src/rest-server/src/models/v2/group.js]: remove `+`, you can use `\` if you do not want a new line

**Comment by ydye** [comment]
@mzmssg You should send another PR to master branch

**Comment by mzmssg** [comment]
> @mzmssg You should send another PR to master branch
After comments resolved

File: src/rest-server/src/models/v2/group.js (hunk 1 above)
Comment (mzmssg) [reply src/rest-server/src/models/v2/group.js]: Seems it's not ECMAScript standards?

**Pull request merged**